.rubocop.yml

```diff
@@ -10,7 +10,7 @@ Metrics/BlockLength:
 Metrics/MethodLength:
   Max: 60
 Layout/LineLength:
-  Max: 250
```

Comment:
I'm curious why you made this change? Generally the guideline is to have it much lower - [The Ruby Style Guide](https://rubystyle.guide/#:~:text=Limit%20lines%20to%2080%20characters.&text=Most%20editors%20and%20IDEs%20have,that%20exceed%20the%20length%20limit.) suggests 80. However this is certainly a debatable topic.